recipe/meta.yaml

```diff
@@ -11,28 +11,35 @@ source:
   sha256: a8616f628ec84353d7a0ba86b228dcf01bab5debc9e4d1a29e5311a52425d594
 
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 1
+  script: {{ PYTHON }} -m build && {{ PYTHON }} -m pip install dist/pdm_backend-{{ version }}-py3-none-any.whl
```

Comment:
Why was this necessary. `pip install` should do this step for you in the background

Reply:
it has been reverted in #25